games-puzzle/tworld/tworld-1.3.2.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Emulation of the game Chip's Challenge"
+DESCRIPTION="Tile World is an emulation of the game Chip's Challenge. \
+Chip's Challenge was originally written for the Atari Lynx by Chuck Sommerville, \
+and was later ported to MS Windows by Microsoft (among other ports). \
+Chip's Challenge is a game made up of both intellectually engaging puzzles \
+and situations demanding fast reflexes. As you might have guessed, \
+it is a tile-based game. The object of each level is simply to get out — i.e., \
+to find and achieve the exit tile. This simple task, however, \
+can sometimes be extremely challenging."
+HOMEPAGE="http://www.muppetlabs.com/~breadbox/software/tworld/"
+COPYRIGHT="2001-2015 Brian Raiter"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI=http://www.muppetlabs.com/~breadbox/pub/software/tworld/tworld-$portVersion-CCLPs.tar.gz"
+CHECKSUM_SHA256="bb867a05e18613260ffad597d8afef8210687495746857a7ad00ef0f2126e12a"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
No need for SECONDARY_ARHCITECTURES (or secondaryArchSuffix following)

Reply:
@Begasus The game doesn't build on x86_gcc2 so I need to put that. BTW, how can I compile this game on x86_gcc2 with secondary arch?

Reply:
Oh.

Reply:
@Begasus Hey, after editing the configure, it becomes unusable
What now! :/

Reply:
What did you change? You only need to remove the "-Wextra" flag, shouldn't interfere with the build (PS, checking if the icon is ok) :)

Reply:
@Begasus I have already checked that. The icon works fine but where do I need to remove the -Wextra?

Reply:
"-Wextra" is only mentioned once in "configure"

Reply:
> "-Wextra" is only mentioned once in "configure"

no

Reply:
Steps to create a recipe (how I do it here):
1. clean your build (if not done already) with "hp -c tworld"
2. extract the source but don't build with "hp -b tworld"
3. edit configure and save it
4. open a Terminal in the sourcetree
5. check your changes with "git status" (should show only 1 file - configure)
6. commit your changes to git with "git commit -a" and provide a message for it, then exit and save the text editor opened by git
7. extract the patchset with: "hp -e tworld" (it will create a new folder "patches" aside of you recipe
8. add PATCHES="*" where * is the (full) name of your patchset (in this case it would produce "tworld-1.3.2.patchset") in your recipe
9. clean your build again with "hp -c tworld"
10. build tworld with "hp tworld"
All this is for primary architecture, if you run for secondary architecture replace "tworld" with "tworld_x86"

Reply:
> > "-Wextra" is only mentioned once in "configure"
> 
> No, it is mentioned like 4 times

My bad, indeed 4 times, all should be removed (only -Wextra)